docs/user-guide/src/SUMMARY.md

```diff
@@ -1,10 +1,26 @@
 # Metal3 Project
 
 [Introduction](introduction.md)
-[Concepts]()
 
 - [Project Overview]()
-- [Install Baremetal Operator]()
-- [Install Cluster-api-provider-metatl3]()
-
-[Community]()
\ No newline at end of file
+- [Baremetal Operator]()
+    - [Install Baremetal Operator]()
+    - [Features]()
+        - [Automated Cleaning API]()
+        - [Automatic Secure Boot]()
+        - [Boot-iso API]()
+        - [Detached Annotation]()
+        - [External Inspection]()
+        - [Inspect annoatation]()
+        - [Reboot annotation]()
+- [Ironic in Metal3]()
+    - [Install Ironic]()
+- [Cluster-api-provider-metal3]()
```

Comment:
Maybe add a section for upgrading? I think it would be useful to point out in what order to upgrade things and if there are any special procedures needed, e.g. remove old version before installing new or something like this. Not specific for CAPM3 but I thought it could fit in this chapter.

Reply:
I agree, upgrade is one of the major things we do with the provider. +1